src/Controller/CrimeSceneController.php

```diff
@@ -34,9 +35,20 @@ public function add(): string
 
     public function show(int $id): string
     {
+        // si je fais une requête de type POST
+
         $crimeSceneManager = new CrimeSceneManager();
         $crimeScene = $crimeSceneManager->selectOneById($id);
 
+        $commentManager = new CommentManager();
+        $commentScene = $commentManager->selectOneByIdWithComment($id);
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            // clean $_POST data
+            $commentScene = array_map('trim', $_POST);
+            // créer un commentaire associé à $id
```

Comment:
$commentScene should be $sceneComment or $newComment, or something like that

the comment is not associated to $id: there is no $newComment['crimescene_id] = $id